apps/auth/package.json

```diff
@@ -36,6 +38,7 @@
     "@types/passport": "^1.0.6",
     "@types/passport-local": "^1.0.33",
     "@types/validator": "^13.1.3",
+    "nodemon": "^2.0.15",
```

Comment:
add it to the devDependencies only.

Reply:
Its just in the devDependencies. isn't it ?

Reply:
yes, you're right. the PR review screen is misleading. :)